crates/services/p2p/src/service.rs

```diff
@@ -995,11 +999,9 @@ where
                     }
                 }
                 self.next_check_time += self.heartbeat_check_interval;
+                TaskNextAction::Continue
             }
         }
-
-        tracing::debug!("P2P task is finished");
```

Comment:
Maybe we can also get rid of the "is running" [debug print](https://github.com/FuelLabs/fuel-core/pull/2429/files#diff-f4a43f3e5d08c9926062c05e8f8e237ba1373bbdf51be7fda0d91da708e5b959L858)?